doc/api/readline.md

```diff
@@ -358,6 +358,9 @@ added: v0.1.98
     only if `terminal` is set to `true` by the user or by an internal `output`
     check, otherwise the history caching mechanism is not initialized at all.
   * `prompt` - the prompt string to use. Default: `'> '`
+  * `crlfDelay` {number} If the delay between `\r` and `\n` exceeds
+    `crlfDelay`, both `\r` and `\n` will be treated as separate end-of-line
```

Comment:
I am not sure about the maximum delay but minimum delay should not be zero. the reason is subsequent events(`\r` followed by `\n`) may have latency greater than or equal to 1ms. It will obviously break windows system. 
What do you suggest ? 10ms as minimum delay?